deploy/kubernetes/Makefile

```diff
@@ -3,7 +3,7 @@ MANIFESTS = $(or $(shell printenv MANIFESTS), manifests)
 
 .PHONY: gen-complete-demo
 gen-complete-demo:
-	cat ${MANIFESTS}/* > complete-demo.yaml
+	awk '{print}' ${MANIFESTS}/* > complete-demo.yaml
 
 .PHONY: check-complete-demo
 check-complete-demo:
```

Comment:
`awk '{print}' ${MANIFESTS}/* | diff complete-demo.yaml -` should do the job here, as it is it breaks in the pipeline checks

Reply:
Done